docs/how-to-use-runhouse.rst

```diff
@@ -0,0 +1,285 @@
+still unproblematic for additional future iteration or debug, since you can easily interactively layer on changes to the environment
+from local, even when you launch with the container.
+
+My Pipeline is in Production, What's Next?
+----------------------
+Once in production, your ML pipelines will eventually experience some failure you need to debug. With Runhouse engineers can easily reproduce production runs on local,
+make changes to the underlying code, and simply push a change to the codebase. There is no debugging through the orchestrator, and no need to rebuild and resubmit.
+However, we find that deploying with Runhouse has fewer errors to begin with, as the code has already been developed in a production-like environment.
+
+This also makes production-to-research a seamless process. Many teams are loathe to revisit the research-to-production process again, and so when code is deployed
+to production, there is little appetite to make small incremental improvements to the pipeline. With Runhouse, the pipeline is already running serverlessly, and so
+incremental changes that are merged to the team codebase are automatically reflected in the production pipeline once tested via normal development processes.
+
+There are other benefits to using Runhouse in production as you scale up usage. A few are included here:
+
+* **Shared services**: You may want to deploy shared services like an embeddings endpoint, and have all pipelines call it by name as a live service *or* import the code
+from the underlying team repository and stand it up separately in each pipeline. Either way, if you every update or improve this shared service,
+all pipelines will receive the downstream updates without any changes to the pipeline code.
+* **Compute abstraction**: As you add new resources to your pool, get credits from new clouds, or get new quota, if all users are using Runhouse to allocate
+ephemeral compute, there is no need to update any code or configuration files at the user level. The new resources are added by the platform team, and then automatically
+adopted by the full team.
+* **Infrastructure Migrations**: With Runhouse, your application code is entirely undecorated Python and the dispatch happens to arbitrary compute. If you ever choose
+to abandon your existing orchestrator, cloud provider, or any other tool, you simply have to move a small amount of dispatch code and infrastructure code configuration.
+* **Adopting Distributed Frameworks**: Runhouse is a perfect complement to distributed frameworks, with some built-in abstractions that let you scale to multiple clusters
+or start using Ray clusters easily.
+
+
+ML Workflow with and without Runhouse
+-------------------------------------
+A quick high-level summary of the differences between developing and deploying ML code with and without Runhouse:
+
+.. list-table::
```

Comment:
i like the visual here with the table layout, maybe we pull another visual above too for demonstrating how the P2R path is also restored with Runhouse?

Reply:
What I couldn't figure out was whether to put this chart onto Index vs. How to Use vs. Other page